### PR TITLE
style(copyright): Updates all header files for copyright.

### LINKED ISSUE #247: change and ensure that copyright info is in every file
we should change 
`Copyright (C) 2006 INRIA Futurs <renaud.pawlak@inria.fr>`
to
`Copyright (C) 2006-2015 INRIA and contributors`

### PR DESCRIPTION
**Pull request by GerardPaligot**
Closes #247

### CODE REVIEW

File: CeCILL-C.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Copyright (C) 2006-2015 INRIA and contributors
```
Comment (monperrus) [inline CeCILL-C.txt]: we already have LICENSE.txt

Comment (GerardPaligot) [inline CeCILL-C.txt]: CeCILL-C.txt contains the template of the license. LICENSE.txt contains the full text of the license. This isn't the same thing but I can rename CeCILL-C.txt to template_license.txt if you want.

Comment (monperrus) [inline CeCILL-C.txt]: What about CeCILL-C-long.txt
<https://github.com/INRIA/spoon/pull/449#discussion_r46989343> and
CeCILL-C-short.txt
<https://github.com/INRIA/spoon/pull/449#discussion_r46989343> and
LICENSE.txt points to CeCILL-C-long.txt?
<https://github.com/INRIA/spoon/pull/449#discussion_r46989343>

Comment (GerardPaligot) [inline CeCILL-C.txt]: `LICENSE.txt` is a standard by some external tools. I don't think it is a good idea to change it. But, I can change `CeCILL-C.txt` by `LICENSE-short.txt` if you want.

Comment (monperrus) [inline CeCILL-C.txt]: > I can change |CeCILL-C.txt| by |LICENSE-short.txt| if you want. 
+1

**Comment by GerardPaligot** [comment]
@monperrus PR ok.

**Comment by monperrus** [comment]
excellent PR for the commit stats :-)

**Pull request merged**